java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/order/EntreeOrdered.java

```diff
@@ -0,0 +1,14 @@
+package com.my.FoodTruckApp.order;
```

Comment:
this should probably be in the entree dir

Reply:
This has been moved